### PR TITLE
Add typescript support

### PR DESCRIPTION
**Pull request by MrHayato**
Added another filter parameter to the ConfigParser to allow .ts files
Added section to configuration docs on setting up TypeScript support
Modified ConfigParser test to check for .ts files

### CODE REVIEW

File: test/spec/unit/configparser.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ describe('ConfigParser', () => {
         configParser.addConfigFile(FIXTURES_PATH + '/exclude.conf.js')
         let specs = configParser.getSpecs()
         specs.should.not.include(__filename)
-        specs.should.include(__dirname + '/call.js')
+        specs.should.include(path.resolve(__dirname + '/call.js'))
+        specs.should.include(path.resolve(__dirname + '/typescript.ts'))
```
Comment (christian-bromann) [inline test/spec/unit/configparser.js]: Could we move this file into the fixtures directory and modify the exclude.conf.js to also point to lets say all ts files in that fixture directory?

Comment (MrHayato) [inline test/spec/unit/configparser.js]: Would you be opposed to an exclude3.conf.js that handles this? I can create a new test that uses exclude3 and revert this one back to the original.

Comment (christian-bromann) [inline test/spec/unit/configparser.js]: nope, not at all

Comment (MrHayato) [inline test/spec/unit/configparser.js]: Actually see my next commit and see if that works for you

**Comment by christian-bromann** [comment]
Sweet thanks!

**Pull request merged**